index.js

```diff
@@ -77,11 +69,12 @@ module.exports = (entryPoint, opts = {}) => {
     return test(name, async t => {
       const _browser = await browser
       const page = await _browser.newPage()
-      if (opts.location) {
-        await page.goto(opts.location)
-      } else {
-        await page.setContent(index)
-      }
+
+      // we use a file url here so that the default page gets loaded in a secure context.
+      // required to test apis like https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto
+      // http://www.chromium.org/Home/chromium-security/security-faq#TOC-Which-origins-are-secure-
+
+      await page.goto(opts.location || 'file://' + index)
```

Comment:
This is a better default, so I'm all for this change, but just be aware that there are still features in Chrome that are ONLY accessible to verified TLS pages.

I have a few tests in other projects where I load https://wikipedia.com and then inject my test code into that page rather than loading a test page because it's the only way to test the features I'm using.